index.html

```diff
@@ -64,15 +64,18 @@ <h1>Bikes for Refugees</h1>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <div class="button2">
+              <button>Donate a bike today</button>
+              <button id="volunteer">Volunteer</button>
+            </div>
+         
           </div>
 
           <div>
             <h2 class="heading-underline">Learn more</h2>
 
-            <div>
-              <div class="article">
```

Comment:
Instead of Div element, it is recommended to use the Semantic HTML element  "Article".